plugins/org.jkiss.dbeaver.ext.oracle.ui/src/org/jkiss/dbeaver/ext/oracle/ui/tools/sqldeveloper/ConfigImportWizardPageSqlDeveloper.java

```diff
@@ -269,9 +292,17 @@ private void parseJsonConnections(File connectionsFile, ImportData importData) t
                     }
                     if (!CommonUtils.isEmpty(info.getRole())) {
                         connectionInfo.setProviderProperty(OracleConstants.PROP_INTERNAL_LOGON, info.getRole());
+                        // dbeaver only supports SYSDBA, SYSOPER, and default auth logon roles
+                        if (info.getRole().equals("SYSDBA") || info.getRole().equals("SYSOPER")) {
```

Comment:
Please use `OracleConnectionRole.SYSDBA.getTitle()` and `OracleConnectionRole.SYSOPER.getTitle()` instead